newscript.go

```diff
@@ -0,0 +1,405 @@
+	err = cmd3.Run()
+	if err != nil {
+		return err
+	}
+	fmt.Printf("--------Getting bb \n")
+/* Removing in favor of bb
+	cmd4 := exec.Command("go", "run", "scripts/ramfs.go")
+	err = cmd4.Run()
+	if err != nil {
+		return err
+	}
+*/
+	if _, err := os.Stat("/tmp/initramgs.linux_amd64.cpio"); err != nil {
+		return err
+	}
+	fmt.Printf("Created the initramfs in /tmp/")
+	return nil
+}
+
+// Get the right kernel for the current config from github (needs work)
+// https://github.com/LaconicPneumonic/linux-4.12.7.git
+func kernelGet() error {
+	fmt.Printf("-------- Getting the kernel \n")
+	cmd8 := exec.Command("git", "clone", "https://github.com/LaconicPneumonic/linux-4.12.7.git")
+	err := cmd8.Run()
+	if err != nil {
+		fmt.Printf("didn't clone kernel from Anthony's repo")
+		return err
+	}
+	return nil
+}
+
+
+
+// Get the right kernel for the current config in NiChrome (Needs work)
+func linuxKernelGet() error {
+	fmt.Printf("-------- Getting the kernel \n")
+	fmt.Printf("Current version of Kernel is %v\n", kernelVersion)
+	name := fmt.Sprintf("%s.tar", linuxVersion)
+	out, err := os.Create(name)
+	fmt.Printf("Kernel saved in %s\n", name)
+	if err != nil {
+		return err
+	}
+	defer out.Close()
+	linkName := fmt.Sprintf("%s%s%s", "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-", kernelVersion, ".tar.xz")
+	fmt.Printf("Pulling from link %s\n", linkName)
+	resp, err := http.Get(linkName)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	_, err = io.Copy(out, resp.Body)
+	if err != nil {
+		return err
+	}
+	cmd5 := exec.Command("tar", "xf", fmt.Sprintf("%s.tar",linuxVersion))
+	err = cmd5.Run()
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// To get the newest Kernel from kernel.org
+func newestkernelGet() error {
+	jsonFile, err := http.Get("http://www.kernel.org/releases.json")
+	if err != nil {
+		return err
+	}
+	defer jsonFile.Body.Close()
+	d, err := ioutil.ReadAll(jsonFile.Body)
+	version := strings.Split(strings.SplitAfter(strings.Split(strings.Split(string(d), "\"moniker\": \"stable\",")[1], "\n")[1], "https:")[1], "\",")[0]
+	fmt.Printf("\n Version of Kernel is %v", version)
+	name := fmt.Sprintf("%s", "linux.tar")
+	out, err := os.Create(name)
+	fmt.Printf("\n Kernel saved in %s", name)
+	if err != nil {
+		return err
+	}
+	defer out.Close()
+	resp, err := http.Get(fmt.Sprintf("http:%s", version))
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	_, err = io.Copy(out, resp.Body)
+	if err != nil {
+		return err
+	}
+
+	cmd5 := exec.Command("tar", "xf", fmt.Sprintf("%s.tar",linuxVersion))
+	err = cmd5.Run()
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+
+func unpackKernel() error {
+	fmt.Printf("-------- Unpack the kernel\n")
+	cmd8 := exec.Command("git", "clone", "https://github.com/u-root/NiChrome.git")
+	err := cmd8.Run()
+	if err != nil {
+		fmt.Printf("didn't clone Nichrome")
+		return err
+	}
+	cmd9 := exec.Command("cp", "NiChrome/CONFIG", fmt.Sprintf("%s/.config", linuxVersion))
```

Comment:
use go code to do this. 
You can use ioutil.ReadFile and ioutil.WriteFile. For every cp command you should be able to do this.
In face, you can write a function, cp, that does this for you, but that's for later.